lua/null-ls/builtins/diagnostics/npm_groovy_lint.lua

```diff
@@ -66,8 +66,23 @@ return h.make_builtin({
     filetypes = { "groovy", "java", "Jenkinsfile" },
     generator_opts = {
         command = "npm-groovy-lint",
-        args = { "-o", "json", "-" },
-        to_stdin = true,
+        args = function(params)
+            local args = {
+                "--failon",
+                "none",
+                "-o",
+                "json",
+                "$FILENAME",
+            }
+            local JENKINSFILE = "Jenkinsfile"
+            if params.bufname:find(JENKINSFILE) or vim.bo.filetype == JENKINSFILE then
```

Comment:
I think the ft should be lowercase? Correct me if I'm wrong.

Reply:
honestly there's no such filetype, I just map them to groovy. what do you suggest here?

Reply:
I think it's sufficient to remove the filetype check here. <https://github.com/neovim/neovim/blob/master/runtime/lua/vim/filetype.lua?plain=1#L1589>

Reply:
removed. I do need the bufname find because I have Jenkinsfiles that their name is something like `my-pipeline.Jenkinsfile`